go.mod

```diff
@@ -8,16 +8,15 @@ replace k8s.io/klog => github.com/istio/klog v0.0.0-20190424230111-fb7481ea8bcf
 
 replace github.com/spf13/viper => github.com/istio/viper v1.3.3-0.20190515210538-2789fed3109c
 
+replace istio.io/operator => github.com/ayj/operator v0.0.0-20190723215839-2311a0e32cec
```

Comment:
Will revert before submit.

Reply:
we use vendor in this repo still so I don't think this will do anything without updating vendor - I could be wrong though. See https://github.com/istio/istio/wiki/Vendor-FAQ

Reply:
Thanks for the pointer. Surprisingly it seems to work locally.